Disable unnecessary warning

Leonardiae opened the pull request:
While syncing my node from scratch this warning first scared me:
`IsBlockPayeeValid -- WARNING: Client synced but old budget system is disabled, accepting any payee`
Then I read on discord that someone else had the same question before and Bigpiggy answered:  "just means were are not running the dash system for budget" 

So I figured why not disable this warning, since WARNING sounds kind of serious.

Maybe this change will help prevents others having the same question about this warning in the future.

Leonardiae committed:
Disable unnecessary warning

While syncing my node from scratch this warning first scared me. Then I read on discord someone else once had the same question and then Bigpiggy wrote:  "just means were are not running the dash system for budget" So I figured then why not disable this warning, since WARNING sounds kind of serious.

Maybe this change will help prevents others having the same question about this warning in the future.

npq7721 reviewed (commented):
i would not remove the comment completely, would just put log on different level so normally u don't see it. so if we decide to have this feature we may be using this log for debugging

Leonardiae committed:
Change loglevel of unnecessary warning

Changed the loglevel instead of completely disabling the warning after review @npq7721

npq7721 reviewed (approved):
LGTM
